<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomateExcel</class>
 <widget class="QDialog" name="AutomateExcel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adjust File Screen</string>
  </property>
  <widget class="QPushButton" name="btnSelectFile">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Commissions File</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtConsole">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>441</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Dubai Medium'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTaarcomImage">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>TAARCOM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbltxtCurrentFile">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Current File:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSelectedFile">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;No File Selected&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearConsole">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>510</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Console</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeselectFile">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>clear-file.png</normaloff>clear-file.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="drpdwnCustomer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string>ALL</string>
   </property>
   <item>
    <property name="text">
     <string>ALL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Top 10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Top 25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Top 50</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnRun">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QComboBox" name="drpdwnPrincipal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string>ALL</string>
   </property>
   <item>
    <property name="text">
     <string>ALL</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbltxtCustomer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Customer(s):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbltxtPrincipal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Principal(s):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbltxtDateColumn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Column:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbltxtTimePeriod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Period:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="drpdwnDateColumn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Paid Date</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Invoice Date</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateStartDate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEndDate">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>7</month>
     <day>6</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="lbltxtThru">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>THRU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
